public/Set-DbaNetworkCertificate.ps1

```diff
@@ -97,23 +97,13 @@ function Set-DbaNetworkCertificate {
             Write-Message -Level VeryVerbose -Message "Processing $instance" -Target $instance
             $null = Test-ElevationRequirement -ComputerName $instance -Continue
 
-            try {
-                Write-Message -Level Verbose -Message "Resolving hostname."
-                $resolved = $null
-                $resolved = Resolve-DbaNetworkName -ComputerName $instance -Credential $Credential -EnableException
-            } catch {
-                $resolved = Resolve-DbaNetworkName -ComputerName $instance -Credential $Credential -Turbo
-            }
-
-            if ($null -eq $resolved) {
-                Stop-Function -Message "Can't resolve $instance" -Target $instance -Continue -Category InvalidArgument
-            }
 
             $computerName = $instance.ComputerName
             $instanceName = $instance.instancename
 
             try {
-                $sqlwmi = Invoke-ManagedComputerCommand -ComputerName $resolved.FQDN -ScriptBlock { $wmi.Services } -Credential $Credential -ErrorAction Stop | Where-Object DisplayName -eq "SQL Server ($instanceName)"
+                # removed: Resolve-DbaNetworkName command as it is used in the Invoke-ManagedComputerCommand anyway
+                $sqlwmi = Invoke-ManagedComputerCommand -ComputerName $computerName -ScriptBlock { $wmi.Services } -Credential $Credential -ErrorAction Stop | Where-Object DisplayName -eq "SQL Server ($instanceName)"
```

Comment:
This change from `$resolved.FQDN` to `$computerName` has to be applied to new lines 232 and 239 as well.

Reply:
@HCRitter Do you have time to update the pull request?